topics/transcriptomics/tutorials/scrna-plant/tutorial.md

```diff
@@ -0,0 +1,533 @@
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata` (output of **Manipulate AnnData** {% icon tool %})
+>    - *"Method used for inspecting"*: `Calculate quality control metrics, using 'pp.calculate_qc_metrics'`
+>
+> 1. {% tool [Plot with scanpy](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_plot/scanpy_plot/1.7.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata` (output of **Manipulate AnnData** {% icon tool %})
+>    - *"Method used for plotting"*: `Generic: Violin plot, using 'pl.violin'`
+>      - *"Keys for accessing variables"*: `Subset of variables in 'adata.var_names' or fields of '.obs'`
+>        - *"Keys for accessing variables"*: `n_genes_by_counts, total_counts`
+>      - *"The key of the observation grouping to consider"*: `batch`
+>      - In *"Violin plot attributes"*:
+>        - *"Add a stripplot on top of the violin plot"*: `Yes`
+>          - *"Add a jitter to the stripplot"*: `Yes`
+>            - *"Size of the jitter points"*: `0.4`
+>        - *"Display keys in multiple panels"*: `Yes`
+>
+{: .hands_on}
+
+This should result in the following violin plot, where we see the distribution of the number of genes per cell (`n_genes_by_counts`) and the library size per cell, where the library size is the total number of mRNA in a cell, regardless of which gene it came from (`total_counts`).
+
+!["Violin Plots per Batch"](../../images/scrna-plant/violin1.png)
+
+Each dot is a cell, and the x-axis has two groups showing the absolute values of for each batch (`shr` and `wt`). The y-values correspond to the actual value of each cell, but the x-vales within each group are just random jitter to help see the cells more clearly. The outline of the violin plot shows us the density of each batch, where we can see that most of the cells in the `shr` group have an average number of features just below 2000 genes, compared to `wt` which has an average number of features just below 4000 genes. If we compare the library size however, we notice that the outlines are very similar and that the average library size across batches are comparable.
+
+### Filter
+
+Most scRNA-seq datasets need to set a minimum threshold of detectability in order to ensure that the cells are of high enough quality to be used for analysis. Conversely, sometimes an upper threshold also should be set to counter the case where a cell is a cell-doublet (in the sense that two cells were captured and sequenced accidentally as a single cell). 
+
+The above violin plots suggest quite a few outliers, mostly on the upper-end of the density plots, suggesting a few cell-doublets in the data. The effect of these is not something we cannot easily regress out, and so we must filter them out instead. By looking at the plots, we can see that a good maximum library size here would be 100 000 transcripts, and a good maximum feature size would be 10 000 transcripts.
```

Comment:
```suggestion
The above violin plots suggest quite a few outliers, mostly on the upper-end of the density plots, suggesting a few cell-doublets in the data. The effect of these is not something we cannot easily regress out, and so we must filter them out instead. By looking at the plots, we can see that a good maximum library size here would be 100 000 transcripts, and a good maximum feature size would be 10000 transcripts.
```

Reply:
So I tend to leave a space, because my eyes suffer otherwise and in the previous part I say 100 000. I think I want to keep this